Makefile

```diff
@@ -8,6 +8,7 @@ SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
 GITDIR        = $(shell git rev-parse --abbrev-ref HEAD)
+STAGINGURL    = http://192.241.195.202:9000/staging
```

Comment:
Is this a globally shared URL? How does it work when multiple people do this at the same time?

Reply:
Differentiated by git branch, got it!